messages/variables/messages.go

```diff
@@ -0,0 +1,16 @@
+package variables
+
+var (
+	VariablesUsage 				= "variables"
+	VariablesShortDescription   = "Manage your variables' varaibles on the Azion Edge platform"
```

Comment:
varaibles? 
I would suggest: "Manage your variables on the Azion Edge platform" for now

Reply:
ok, i will fix it